packages/commons/src/version.ts

```diff
@@ -0,0 +1,2 @@
+// this file is auto generated, do not modify
+module.exports = { version: '1.10.0' };
```

Comment:
```suggestion
export const PT_VERSION = '1.10.0';
```

Reply:
This way we are not mixing exports type, the one with `module.exports` is CJS only.

Also if you export it with the final name, you don't have to alias it with `as PT_VERSION` when you import it, same for the object vs `string`

Reply:
agree